playwright-e2e/data/fake-user.json

```diff
@@ -51,5 +51,51 @@
     "city": "Cambridge",
     "zip": "02142-1027",
     "phone": "6177147000"
+  },
+  "secondChild": {
+    "firstName": "E2E-second-Child",
+    "middleName": "Junior",
+    "lastName": "Playwright",
+    "birthDate": {
+      "MM": "07",
+      "DD": "20",
+      "YYYY": "2016"
+    },
+    "age": "06",
```

Comment:
Any age under 7, right? I think it'd be good to make child age smaller than 6 so this new test won't fail next year.

Reply:
done